usaspending_api/recipient/v2/lookups.py

```diff
@@ -1,3 +1,12 @@
+# based from all_award_types_mappings in usaspending_api.awards.v2.lookups.lookups
+AWARD_TYPES = {
+    "contracts": {"amount": "last_12_contracts", "filter": "contract"},
+    "grants": {"amount": "last_12_grants", "filter": "grant"},
+    "direct_payments": {"amount": "last_12_direct_payments", "filter": "direct payment"},
+    "loans": {"amount": "last_12_loans", "filter": "loans"},
+    "other_financial_assistance": {"amount": "last_12_other", "filter": "other"},
+}
```

Comment:
Not sure where these are used and ripple effect, but I suggest keeping pluralization consistent between col name and filter name. 

e.g. 
- `last_12_contracts` -> `contract` (not pluralized, same for `grant`)
- `last_12_loans` -> `loans` (pluralized here)

Reply:
If the change were to break existing api contracts though, you may not want to do it (it would require keeping backwards compatibility or going to a new v3 for this endpoint).

Reply:
Making this change would require updates to the frontend as well

Reply:
no need then